truco-careca-server/src/game.rs

```diff
@@ -38,19 +37,25 @@ impl Game {
         self.list_players.len() == 4
     }
 
-    pub fn get_players(&mut self) -> &HashMap<u32, (Connection, Player)> {
-        &self.list_players
```

Comment:
`list_players` is not public, we prob gonna need this method some time in the future

Reply:
I do agree that if we don't need that for now we delete it, I was using this to print to std out in the main.rs

Reply:
just merge this!